indico/MaKaC/badgeDesignConf.py

```diff
@@ -77,15 +77,32 @@ def __init__(self):
 
         self.items_actions = {
             "Title": (_("Title"), lambda x: x.get_personal_data().get('title', '')),
-            "Full Name": (_("Full Name"), lambda x: "{} {} {}".format(x.get_personal_data().get('title', ''),
-                                                                      x.last_name, x.first_name)),
-            "Full Name (w/o title)": (_("Full Name (w/o title)"), lambda x: "{} {}".format(x.last_name, x.first_name)),
-            "Full Name B": (_("Full Name B"), lambda x: "{} {} {}".format(x.get_personal_data().get('title', ''),
-                                                                          x.first_name, x.last_name)),
-            "Full Name B (w/o title)": (_("Full Name B (w/o title)"), lambda x: "{} {}".format(x.first_name,
-                                                                                               x.last_name)),
-            "First Name": (_("First Name"), lambda x: x.first_name),
-            "Surname": (_("Surname"), lambda x: x.last_name),
+            "Full Name": (_("Full Name"), lambda x: "{} {}".format(x.get_personal_data().get('title', ''),
```

Comment:
You UTF8-encode some  things (names) but not others (address for example). Why not get rid of the encoding here altogether and apply it in the end (i.e. after calling the lambda)?